Resources/Prototypes/Nyanotrasen/Catalog/Fills/Lockers/ourjobs.yml

```diff
@@ -13,6 +13,7 @@
       - id: DoorRemoteResearch
       - id: HandTeleporter
       - id: RubberStampRd
+      - id: BoxEncryptionKeyScience
```

Comment:
Changing little things like this is usually best left to the Upstream that made it, as to not cause conflicts when pulling. I imagine Nyano has already gotten to this in the updates we haven't merged.
That said, I'm fine with leaving it for now. Just thought I'd let you know for future.

Reply:
I'll make sure to keep that in mind. I'll revert it for now.